01_fib/alanhillgemann/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+func fib(n int) {
+	if n < 0 {
+		log.SetFlags(0)
+		log.Printf("Fib of %d is not available. Defaulting to minimum fib of 0", n)
+		n = 0
+	} else if n > 45 {
+		log.SetFlags(0)
+		log.Printf("Fib of %d is not available. Defaulting to maximum fib of 45", n)
+		n = 45
+	}
+	fibSlice := []int{}
```

Comment:
You already know how big you want your slice to be - create it with make `make([]int, n)` and use indices instead of append or provide capacity `make([]int, 0, n)`

Reply:
Fixed. Array removed